vendor/github.com/ciscoecosystem/mso-go-client/client/client.go

```diff
@@ -74,6 +75,12 @@ func Platform(platform string) Option {
 	}
 }
 
+func SkipLoggingPayload(skipLoggingPayload bool) Option {
```

Comment:
what is this func for?

Reply:
It is used to set the value of the variable. It is just an accessor. I copied it from ACI code.